wlauto/workloads/googlephotos/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -27,24 +28,15 @@
     public void runUiAutomation() throws Exception {
         parameters = getParams();
 
-        confirmLocalFileAccess();
         dismissWelcomeView();
         gesturesTest();
-        editPhotoTest();
+        editPhotoColorTest();
+        cropPhotoTest();
+        rotatePhotoTest();
 
         writeResultsToFile(timingResults, parameters.getString("output_file"));
     }
 
-    private void confirmLocalFileAccess() throws Exception {
-        // First time run requires confirmation to allow access to local files
-        UiObject allowButton = new UiObject(new UiSelector().textContains("Allow")
-                                                            .className("android.widget.Button"));
-        if (allowButton.waitForExists(timeout)) {
-            allowButton.clickAndWaitForNewWindow(timeout);
-        }
-    }
-
-
```

Comment:
Can this not be set in the settings menu on the device for each app? This causes the workload to crash when I run it on my devices. I see that the photo app has access to storage in the device app settings.